nvflare/private/fed/server/job_runner.py

```diff
@@ -464,6 +465,7 @@ def run(self, fl_ctx: FLContext):
                             with self.lock:
                                 self.running_jobs[job_id] = ready_job
                             job_manager.set_status(ready_job.job_id, RunStatus.RUNNING, fl_ctx)
+                            self.log_info(fl_ctx, f"Job: {job_id} started to run, status changed to RUNNING.")
```

Comment:
Can you add more messages:
- right before self._start_run()
- after self._start_run() and before "with self.lock"
- before job_manager.set_status(ready_job.job_id, RunStatus.RUNNING, fl_ctx)

This can help better understand where it's stuck.

Reply:
Added more logging.